airbyte-integrations/connectors/source-shopify/metadata.yaml

```diff
@@ -17,6 +17,7 @@ data:
   githubIssueLabel: source-shopify
   icon: shopify.svg
   license: ELv2
+  maxSecondsBetweenMessages: 1
```

Comment:
@bazarnov does it make sense in terms of rate limit for shopify (i.e. the rate limit window is one second)? @davinchia mentioned that there would still be a minimum. @davinchia, can you confirm the behavior when the `maxSecondsBetweenMessages` is lower than 90 minutes?

Reply:
@maxi297 It doesn't regard to the BULK Jobs running in avg 150 - 600 sec. I would rather reference the Shopify Documentation, where this info could have been, but there is no mentioned of how long the regular bulk job would take to run. My opinion here is we should rely on the value of 2 hours, and make it possible to adjust the slice size from the source itself, so the source logic follows the timings as well, when it's possible (if we have long-running jobs - we decrease the next slice to try-run faster, and if we have faster runs for bulk jobs, we can increase the slice size to try get bigger data portion next time, thus dynamically balance the slice size - working on it, btw)

so, @strosek it should be at least: `7200` for `source-shopify` and `86400` for `source-salesforce`, since the max bulk job time for `salesforce` is up to 1 day.

Reply:
So, we've been told by @davinchia  that this value should be the value of the rate limit hence why I asked him to confirm that. Given the naming of the variable, your reasoning sounds good but this was not the ask given to us. @davinchia, please confirm

Reply:
@davinchia confirmed @bazarnov point. Can we update shopify value? I'll approve so that once the change is done, you can merge.